src/theme.js

```diff
@@ -0,0 +1,15 @@
+const theme = {
+  font: {
+    large: "32px",
+    medium: "24px",
+    small: "16px",
+    footer: "12px",
+  },
+  title: "40px",
+  percentColor: "#452CDD",
+  footerColor: "#888888",
+  outerMargin: "76px",
+  cardMargin: "24px",
```

Comment:
margin 값을 이렇게 관리할 수 있을 것 같아요.

```suggestion
  spacing06: "24px",
```

참고: 2의 배수로 스케일 값을 정한 경우
 https://carbondesignsystem.com/guidelines/spacing/overview/

Reply:
```js
const theme = {
  font: {
    large: "2rem",
    medium: "1.5rem",
    small: "1rem",
    footer: "0.75rem",
  },
  title: "40px",
  color: {
    blue: "#452CDD",
    gray: "#888888",
  },
  margin: {
    spacing11: "80px", //outerMargin
    spacing06: "24px", //cardMargin
  },
};
```

- Theme의 형태를 변경했습니다. 
- GlobalStyle의 #root에 base font-size를 지정한 후 모드 상대 수치인 rem으로 변경했습니다.
- margin 관리를 spacing06, spacing11로 변경했습니다. 이는 해당 사이트를 참고하여 변경했습니다. [링크](https://carbondesignsystem.com/guidelines/spacing/overview)